GVFS/GVFS.FunctionalTests/Tests/EnlistmentPerFixture/GitBlockCommandsTests.cs

```diff
@@ -0,0 +1,106 @@
+﻿using GVFS.FunctionalTests.FileSystemRunners;
+using GVFS.FunctionalTests.Should;
+using GVFS.FunctionalTests.Tools;
+using GVFS.Tests.Should;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Runtime.InteropServices;
+
+namespace GVFS.FunctionalTests.Tests.EnlistmentPerFixture
+{
+    [TestFixtureSource(typeof(FileSystemRunner), nameof(FileSystemRunner.Runners))]

```

Comment:
There is no need to parameterize these tests on `FileSystemRunner`, as they never use `this.fileSystem`.

`[TestFixtureSource(typeof(FileSystemRunner), nameof(FileSystemRunner.Runners))]` can be replaced with `[TestFixture]` and the `FileSystemRunner fileSystem` parameter can be removed from the constructor.

Reply:
@benpeart I see I was too late with this comment, I will create a PR with this change.